test/libsolidity/syntaxTests/experimental/inference/callgraph.sol

```diff
@@ -0,0 +1,101 @@
+pragma experimental solidity;
+
+type uint256 = word;
+
+instantiation uint256: + {
+    function add(x, y) -> uint256 {
+        let a = uint256.rep(x);
+        let b = uint256.rep(y);
+        assembly {
+            a := add(a,b)
+        }
+        return uint256.abs(a);
+    }
+}
+
+function unreferenced(x:uint256) -> uint256
+{
+    return x;
+}
+
+function f(x:uint256) -> uint256
+{
+    return g(h(x));
+}
+
+function g(x:uint256) -> uint256
+{
+    return x;
+}
+
+function h(x:uint256) -> uint256
+{
+    return x;
+}
+
+contract C {
+    fallback() external {
+        let a: uint256->uint256 = f;
+    }
+}
+// ====
+// EVMVersion: >=constantinople
+// ----
+// Warning 2264: (0-29): Experimental features are turned on. Do not use experimental features on live deployments.
+// Info 4164: (31-51): Inferred type: uint256:(type, +)
```

Comment:
Uh, should also probably disable debug warner by default, since it makes syntax tests extremely verbose, and not everything needs, or should be a semantic test.